R/profile.R

```diff
@@ -122,10 +122,10 @@ step_profile <- function(recipe,
       i = "See {.help [?step_profile](recipes::step_profile)} for information."
     ))
   }
-  if (all(sort(names(grid)) == c("len", "ptcl"))) {
+  if (any(sort(names(grid)) != c("len", "pctl"))) {
```

Comment:
```suggestion
  if (!identical(sort(names(grid)), c("len", "pctl"))) {
```

This threw me for a moment